core/vm/contracts_stateful.go

```diff
@@ -220,13 +219,15 @@ func (c *exportAsset) Run(evm *EVM, caller ContractRef, addr common.Address, inp
 		return nil, remainingGas, ErrExecutionReverted
 	}
 
-	// create an export transaction to export the asset to the recipient address
-	var keys []*avaCrypto.PrivateKeySECP256K1R
+	// [assetID] is a string encoded asset ID. It will be parsed as either the string literal "AVAX" or by using ids.FromString
```

Comment:
Please read here for a description of arguments. I need validation on whether this is correct or not.